### ISSUE TITLE
Problem with full installation of mingw-x64-x86_64-toolchain

### ISSUE THREAD

**Issue by guteksan** [Alexpux/MINGW-packages#1887]
I am trying to install mingw-x64-x86_64-toolchain. Most of the packages have been installed, but some fail:
```
mingw-w64-x86_64-zlib-1.2.8-10-any                          160.6 KiB  78.4M/s 00:00 [#################################################] 100%
mingw-w64-x86_64-binutils-2.27-2-any                         12.8 MiB   302K/s 00:43 [#################################################] 100%
error: failed retrieving file 'mingw-w64-x86_64-headers-git-5.0.0.4747.0f8f626-1-any.pkg.tar.xz' from repo.msys2.org : Operation too slow. Less than 1 bytes/sec transferred the last 10 seconds
error: failed retrieving file 'mingw-w64-x86_64-headers-git-5.0.0.4747.0f8f626-1-any.pkg.tar.xz' from downloads.sourceforge.net : Operation too slow. Less than 1 bytes/sec transferred the last 10 seconds
error: failed retrieving file 'mingw-w64-x86_64-headers-git-5.0.0.4747.0f8f626-1-any.pkg.tar.xz' from www2.futureware.at : The requested URL returned error: 404
```
some failed at repo.msys2.org, but apparently succeeded at other mirrors. After all, the problem seems to apply to:
```
mingw-w64-x86_64-crt-git-5.0.0.4745
mingw-w64-x86_64-headers-git-5.0.0.4747
```
packages only.
There is no problem however, to download these files directly from: http://repo.msys2.org/mingw/x86_64/

**Comment by peterbud** [comment]
It seems to me rather a network error between your machine and the repo. Is it something you can reproduce always?

**Comment by guteksan** [comment]
Yes, I have atempted to download a couple of times, at different days, and I always failed at these two packages. Also, I had no problem to download them from the repo via a web browser.

**Comment by mati865** [comment]
Looks like loss of packets during download (unstable connection).

**Comment by looeee** [comment]
I am having a similar problem, running various `ridk install` commands:

`ridk install 1`:

```js
Verify integrity of msys2-x86_64-20161025.exe ... Failed
Download http://repo.msys2.org/distrib/x86_64/msys2-x86_64-20161025.exe
  to C:\Users\looee\AppData\Local\Temp/msys2-x86_64-20161025.exe
Downloading msys2-x86_64-20161025.exe (  0%)
Verify integrity of msys2-x86_64-20161025.exe ... Failed
Download http://repo.msys2.org/distrib/x86_64/msys2-x86_64-20161025.exe
  to C:\Users\looee\AppData\Local\Temp/msys2-x86_64-20161025.exe
Downloading msys2-x86_64-20161025.exe (  0%)
Verify integrity of msys2-x86_64-20161025.exe ... Failed
Download http://repo.msys2.org/distrib/x86_64/msys2-x86_64-20161025.exe
  to C:\Users\looee\AppData\Local\Temp/msys2-x86_64-20161025.exe
```

Tried downloading `http://repo.msys2.org/distrib/x86_64/msys2-x86_64-20161025.exe` manually and I was getting speeds of <1kb/s. 

After installing from a different location  and running `ridk install 2` - many errors such as 

```js
error: failed retrieving file 'pacman-5.0.1-4-x86_64.pkg.tar.xz' from repo.msys2.org : transfer closed with 7036272 bytes remaining to read
```

My internet connection is fine (speedtest around 30mbs and downloads from other sites work fine). 
Could this be a regional thing, since I am based in Vietnam?

**Comment by lazka** [comment]
Is this still a problem?
